src/main/java/com/esri/samples/scene/get_elevation_at_a_point/README.md

```diff
@@ -0,0 +1,37 @@
+# Get elevation at a point
```

Comment:
Unfortunately, Java currently requires this to be HTML to display correctly in the sample viewer. You can use https://www.browserling.com/tools/markdown-to-html to do the conversion for you. Should only require a couple small manual steps. Refer to the other samples' readmes.